homeassistant/components/kmtronic/config_flow.py

```diff
@@ -10,11 +10,14 @@
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
 from homeassistant.helpers import aiohttp_client
 
+from .const import CONF_REVERSE
 from .const import DOMAIN  # pylint:disable=unused-import
 
 _LOGGER = logging.getLogger(__name__)
 
-DATA_SCHEMA = vol.Schema({CONF_HOST: str, CONF_USERNAME: str, CONF_PASSWORD: str})
+DATA_SCHEMA = vol.Schema(
+    {CONF_HOST: str, CONF_USERNAME: str, CONF_PASSWORD: str, CONF_REVERSE: bool}
```

Comment:
Are all the options really optional? There are no defaults.

Should the reverse option be part of an options flow instead. Options that are not required to establish the connection we usually put in the options flow to keep the main flow as simple as possible and use a default.

Reply:
Sorry, misunderstanding on my part of config flow data_schema

Reply:
options flow: I did not foresee the need for this option and since got the idea to enable per relay reverse option (just because it is possible), but if you agree that would be a new PR, just want to close this user issue first.

Reply:
I think we should put the reverse option in an options flow if it's not required for establishing the connection.

Reply:
Moved reverse option from initial configuration into options flow